http/src/main/resources/jupyter/startup.sh

```diff
@@ -28,8 +28,9 @@ if [ "$DEPLOY_WELDER" == "true" ] ; then
     gcloud auth configure-docker
     docker-compose -f /etc/welder-docker-compose.yaml up -d
 
-    # Move existing notebooks to new notebooks dir
```

Comment:
I decided to remove this `mv` command:
1. It's scary to `mv` users' files
2. Terra UI re-localizes everything anyway
3. The directory it `mv`s to isn't even correct (it doesn't use the `edit` or `safe` leaf paths)